Counter/src/app/app.component.html

```diff
@@ -1,5 +1,8 @@
-<h1>Shiny Counter</h1>
-
-<p>A simple test.</p>
-<button routerLink="/detail">clicky</button>
+<div class="headerDiv">
```

Comment:
class names are generally kebab-cased so instead of `headerDiv` you would have `header-div` but in this case div isn't really valuable and would be confusing if the element needed to change from a div to a span or h1. Instead, name classes (and ids) based on their purpose on the page and not their underlying styles. In this case, `header` or `app-header` would be fine.

Reply:
Fixed!